purview/models/distgit.py

```diff
@@ -0,0 +1,26 @@
+# SPDX-License-Identifier: GPL-3.0+
+
+from neomodel import (
+    StructuredNode, UniqueIdProperty, RelationshipTo, StringProperty,
+    DateTimeProperty)
+
+
+class DistgitCommit(StructuredNode):
+    id_ = UniqueIdProperty(db_property='id')  # commit_id
+    owner = RelationshipTo('.user.User', 'OWNED_BY')  # author
+    committer = RelationshipTo('.user.User', 'OWNED_BY')
+    pushes = RelationshipTo('DistgitPush', 'PUSHED_BY')
```

Comment:
@yashvardhannanavati  and @ralphbean what are your thoughts on the wording here for the relationship?

Reply:
Hm, use `PUSHED_IN` since "by" makes me thing of a person who pushed.